docs/ddl-of-carbondata.md

```diff
@@ -694,7 +694,7 @@ CarbonData DDL statements are documented here,which includes:
      ```
      ALTER TABLE carbon ADD COLUMNS (a1 INT, b1 STRING) TBLPROPERTIES('DEFAULT.VALUE.a1'='10')
      ```
-      **NOTE:** Add Complex datatype columns is not supported.
+      **NOTE:** Adding of only single-level Complex datatype columns(only array and struct) is supported.
```

Comment:
please give example if adding complex column array and struct

Reply:
done